libzeth/test/circuits/sha256_test.cpp

```diff
@@ -2,23 +2,22 @@
 //
 // SPDX-License-Identifier: LGPL-3.0+
 
-#include "circuits/circuit_types.hpp"
-#include "circuits/circuit_utils.hpp"
-#include "circuits/sha256/sha256_ethereum.hpp"
-#include "types/bits.tcc"
-#include "util.hpp"
+#include "core/bits.hpp"
+#include "libzeth/circuits/circuit_types.hpp"
+#include "libzeth/circuits/circuit_utils.hpp"
+#include "libzeth/circuits/sha256/sha256_ethereum.hpp"
 
 #include <gtest/gtest.h>
 #include <libsnark/common/data_structures/merkle_tree.hpp>
 
 using namespace libsnark;
-using namespace libzeth;
+// using namespace libzeth;
```

Comment:
Let's remove this comment if we do not want to use the namespace here.
Moreover, let's make this consistent across test files as for now some are using the namespace and some aren't

Reply:
I am happy to remove `using libzeth`. I tend to prefer this approach anyway (I really do not like overusing `using <namespace>` since you lose track of where the function comes from). That being said I do not have a strong preference for this specific scenario: both approaches are "equally good to me" for small test files like these ones. So let's pick one and stick to it consistently

Reply:
Removed the comment.